generate/unix/Makefile.config

```diff
@@ -148,6 +148,15 @@ ACPICA_HEADERS = \
 OPT_CFLAGS ?= $(CWARNINGFLAGS)
 
 #
+# Common compiler flags
+# The _GNU_SOURCE symbol is required for many hosts.
+#
+ifeq ($(M32),TRUE)
+CFLAGS +=-m32
+LDFLAGS +=-m32
+endif
```

Comment:
Normally, LDFLAGS can derive from CFLAGS on most compilers.
I'm not sure if ACPICA Makefile is written in this way.
If so, you only need to add CLFAGS here.

Just a reminder.